src/main/service/listManipulation/CivilFamilyAndMixedListService.ts

```diff
@@ -15,11 +15,39 @@ export class CivilFamilyAndMixedListService {
      * Manipulate the family/mixed json data for rendering to the screen.
      * @param list
      */
-    public sculptedFamilyMixedListData(list: string): object {
+    public sculptedFamilyMixedListData(list: string, partyAtHearingLevel = false): object {
+        if (partyAtHearingLevel) {
+            return this.sculptedListDataV1(list, true);
+        }
         return this.sculptedListData(list, true);
     }
 
     private sculptedListData(list: string, isFamilyMixedList = false): object {
```

Comment:
I think this method / equivilant V1 below would be better replaced with something like sculptedListDataHearingLevel etc, just to make it clear. What do you think?

Reply:
Renamed to sculptedListDataPartyAtHearingLevel()